exercises/04_power/power.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import "fmt"
+
+func Pow(x, y float64) float64 {
```

Comment:
1) It should have been a function that operates on integral values (`uint` more specific). 
2) This is an overly complicated solution for integer inputs and overly simplified for floating-point inputs